DirectProgramming/C++SYCL_FPGA/Tutorials/DesignPatterns/onchip_memory_cache/README.md

```diff
@@ -1,29 +1,41 @@
-# Caching On-Chip Memory to Improve Loop Performance
+# `On-Chip Memory Cache` Sample
+
 This FPGA tutorial demonstrates how to build a simple cache (implemented in FPGA registers) to store recently-accessed memory locations so that the compiler can achieve II=1 on critical loops in task kernels.
 
+| Area                 | Description
+|:--                   |:--
+| What you will learn  | How and when to implement the on-chip memory cache optimization
+| Time to complete     | 30 minutes
+| Category             | Code Optimization
+
+## Purpose
+
+This sample demonstrates the following concepts:
+
+- How to implement the on-chip memory cache optimization technique
+- The scenarios in which this technique benefits performance
+- How to tune the cache depth
+
+## Prerequisites
+
 | Optimized for                     | Description
 ---                                 |---
-| OS                                | Linux* Ubuntu* 18.04/20.04 <br> RHEL*/CentOS* 8 <br> SUSE* 15 <br> Windows* 10
-| Hardware                          | Intel® Agilex™, Arria® 10, and Stratix® 10 FPGAs
+| OS                                | Ubuntu* 18.04/20.04 <br> RHEL*/CentOS* 8 <br> SUSE* 15 <br> Windows* 10
+| Hardware                          | Intel® Agilex®, Arria® 10, and Stratix® 10 FPGAs
```

Comment:
```suggestion
| Hardware                          | Intel® Agilex™, Arria® 10, and Stratix® 10 FPGAs
```
I think that it's actualy ™ in the database

Reply:
Interesting, it seems to have changed recently... I checked in this ™ a few weeks ago, but now it appears to be ®...
I'll make the change to all the other FPGA code samples.